Formula/aerc.rb

```diff
@@ -0,0 +1,19 @@
+class Aerc < Formula
+  desc "Email client that runs in your terminal"
+  homepage "https://aerc-mail.org/"
+  url "https://git.sr.ht/~sircmpwn/aerc/archive/0.5.1.tar.gz"
+  sha256 "8feb7f87aa210ab7d8b271c34ad39af13f553447994341b3c25e7a659ce37d46"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "scdoc" => :build
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/aerc", "-v"
```

Comment:
looks like this is still version check?

Reply:
i cant find anything else i can check for... cause aerc only takes "-v" as argument... else it will start the gui...

any ideas?

Reply:
got you, sounds good then. Yeah, GUI test is little bit tricky.